data_models/README.md

```diff
@@ -136,8 +136,48 @@ The fields will be used for normalizing the county case and death tabulations, a
         }
     }
 ```
-5. __Hospitalization Data__  
-We plan to retrieve hospitalization data by county from a common source, the [California Health and Human Services Open Data Portal (CHHS)](https://data.chhs.ca.gov/dataset/california-covid-19-hospital-data-and-case-statistics/resource/6cd8d424-dfaa-4bdd-9410-a3d656e1176e?inner_span=True.). These datapoints are not currently reflected in the data model.
+5. __Hospitalization Data__
+California COVID-19 hospitalization data is retrieved separately from the the [California Health and Human Services Open Data Portal (CHHS)](https://data.ca.gov/dataset/covid-19-hospital-data#). The keys for the top-level data points `name`, `update_time`, `source_url`, `meta_from_source`, and `meta_from_baypd` are the same as the model described above. However, the structure of the data for `meta_from_source` and `series` differs.
+
+`meta_from_source` holds a list of dicts describing each field in the data, provided directly from CHHS. Each dict, except for the first describing the `_id` field, has the keys `info`, `type`, and `id`. Here is an example:
+
+```
+    "meta_from_source": [
+        {
+            "type": "int",
+            "id": "_id"
+        },
+        {
+            "info": {
+                "notes": "The County where the hospital is located. None of the consolidated reporters had hospitals in different counties.",
+                "type_override": "",
+                "label": "County"
+            },
+            "type": "text",
+            "id": "county"
+        }
+
+        -- snip --
+```
+
+Each entry in `series` is a flat record with each of the fields described in `meta_from_source`, one for each day on which an observation was made:
+
+```
+        {
+            "icu_covid_confirmed_patients": 0.0,
+            "icu_suspected_covid_patients": 0.0,
+            "hospitalized_covid_patients": null,
+            "hospitalized_suspected_covid_patients": 10.0,
+            "icu_available_beds": 18.0,
+            "county": "Humboldt",
+            "hospitalized_covid_confirmed_patients": 1.0,
+            "_id": 1,
+            "all_hospital_beds": null,
+            "todays_date": "2020-03-29T00:00:00"
```

Comment:
Over in [`data_models/data_model.json`](https://github.com/sfbrigade/data-covid19-sfbayarea/blob/master/data_models/data_model.json) for each timeseries, we are just storing the date part of the timestamp. I think it makes sense to do that here since it appears the data resolution is only to do the day(`T00:00:00`). Please truncate the timestamp to the date. 
 
(Are these are the correct report dates PDT given the timezone of the database? I don't know anything about timezones except to beware, so forgive my paranoia!)

Finally minor issue: I'd like to change the `"todays_date"` field name to `"report_date"`. It's clear from the metadata, but I still felt some cognitive effort scanning through the data and seeing all these dates that are not today's date! Feel free to ignore if you think it's clear enough.

Reply:
> Over in [`data_models/data_model.json`](https://github.com/sfbrigade/data-covid19-sfbayarea/blob/master/data_models/data_model.json) for each timeseries, we are just storing the date part of the timestamp. I think it makes sense to do that here since it appears the data resolution is only to do the day(`T00:00:00`). Please truncate the timestamp to the date.

Got it - will do.

> (Are these are the correct report dates PDT given the timezone of the database? I don't know anything about timezones except to beware, so forgive my paranoia!)

I can only assume that the dates in the series reflect the date on Pacific time, given that this data is for California. On the landing page for this data, the "Last Updated" time is given in UTC-07:00, so that seems like a safe assumption.

> Finally minor issue: I'd like to change the `"todays_date"` field name to `"report_date"`. It's clear from the metadata, but I still felt some cognitive effort scanning through the data and seeing all these dates that are not today's date! Feel free to ignore if you think it's clear enough.

That makes sense. I'll rename the key.